pwa/src/templates/templateParts/landingDisplayTemplates/categories/CategoriesLandingDisplayTemplate.tsx

```diff
@@ -110,9 +111,9 @@ export const CategoriesLandingDisplayTemplate: React.FC<CategoriesLandingDisplay
         ))}
       </div>
 
-      <Button icon={<ArrowRightIcon />} iconAlign="start" onClick={() => navigate("/categories")}>
-        Bekijk alle categorieën
-      </Button>
+      <ButtonLink onClick={() => navigate("/categories")} appearance="secondary-action-button">
+        <ArrowRightIcon /> Bekijk alle categorieën
+      </ButtonLink>
```

Comment:
This works as intended. However, I'm thinking: should we be passing the `href` prop for accessibility? 

I'm unsure, maybe check this with @Robbert.